modules/datawrkzBidAdapter.js

```diff
@@ -0,0 +1,633 @@
+import * as utils from '../src/utils.js';
```

Comment:
Can you please import just the needed functions please 🙏

Reply:
Pull updated